src/Adapter/Order/OrderProductQuantityUpdater.php

```diff
@@ -505,23 +503,15 @@ protected function reinjectQuantity(
     }
 
     /**
-     * @param Order $order
-     * @param OrderDetail $orderDetail
-     * @param int $oldQuantity
+     * @param int $id_customization
      *
      * @throws OrderException
      */
-    private function updateCustomizationOnProductDelete(Order $order, OrderDetail $orderDetail, int $oldQuantity): void
+    private function deleteProductCustomization($id_customization): void
```

Comment:
```suggestion
    private function deleteProductCustomization(int $id_customization): void
```

Reply:
Resolved in https://github.com/PrestaShop/PrestaShop/pull/12422/commits/517ebc0c1e4aeec3b6053e28d45757db27194f0b